packages/es-dev-server/README.md

```diff
@@ -6,6 +6,12 @@ tags:
   - guides
 ---
 
+> **Notice**
+>
+> Development of es-dev-server continues under a new name: [web dev server](https://modern-web.dev/docs/dev-server/overview/). We recommend using it for new projects, and upgrading for existing projects.
```

Comment:
This link doesn't work, at least against the live site.

Reply:
Hmmm, it seems to work fine for me?